File: typical90/022.cpp
#include <bits/stdc++.h>
using namespace std;
using ll = long long;

int main() {
  ll a, b, c;
  cin >> a >> b >> c;
  ll g = gcd(a, gcd(b,c));
  ll ans = (a/g-1) + (b/g-1) + (c/g-1);
  cout << ans << endl;
  return 0;
}

File: typical90/Rust/022.rs
use proconio::input;
use num::integer::gcd;

fn main() {
    input! { a: usize, b: usize, c: usize };
    let g = gcd(a, gcd(b, c));
    let ans = (a / g) + (b / g) + (c / g) - 3;
    println!("{}", ans);
}
